tools/deployment-cli-tools/ch_cli_tools/utils.py

```diff
@@ -392,6 +392,41 @@ def check_docker_manifest_exists(registry, image_name, tag, registry_secret=None
     resp = requests.get(api_url)
     return resp.status_code == 200
 
+
+def filter_empty_strings(value):
+    return value != ""
+
+
+def search_word_in_file(file, word):
+    p = subprocess.Popen('grep -l %s %s'% (word, file), shell=True,
```

Comment:
This is not pythonic. I'd rather read the file and use regular expression matching